crates/node/engine/src/client.rs

```diff
@@ -2,7 +2,6 @@
 
 use alloy_eips::eip1898::BlockNumberOrTag;
 use alloy_network::AnyNetwork;
-use alloy_network_primitives::BlockTransactionsKind;
```

Comment:
Since this was removed, we need to remove the `alloy_network_primitives` from the cargo manifest so unused crate linting passes